### DISCUSSION TITLE
Secret Detection Addition: HuggingFace Access Tokens

### DISCUSSION THREAD

**Discussion by chrisking** [aquasecurity/trivy#6235] [Ideas]
### Description

HuggingFace is the defecto leader in publishing and distributing models on the internet. User access to the platform is often configured through an access token that can be embedded in code like any other secret. 

More information on User Access Tokens is here: https://huggingface.co/docs/hub/en/security-tokens

It would be great if Trivy detected these tokens and reported them.

### Target

None

### Scanner

Secret

**Comment by knqyf263** [comment]
Created https://github.com/aquasecurity/trivy/issues/6262
